src/Symfony/Component/Finder/SplFileInfo.php

```diff
@@ -28,7 +28,7 @@ class SplFileInfo extends \SplFileInfo
      */
     public function __construct($file, $relativePath, $relativePathname)
     {
-        parent::__construct($file);
+        parent::__construct(realpath($file));
```

Comment:
`realpath($file) ?: $file`?

Reply:
done.

Reply:
realpath() if $file is symlik will return path to original path and if try to remove symlink $file it will be remove original file. so question: how now remove only sumlink without original path?

Reply:
`realpath` usage was reverted already ;)